index.js

```diff
@@ -0,0 +1,5 @@
+const app = require('./server/server');
```

Comment:
maybe we can still use import here?

Reply:
I think that's not possible, because this is nodejs code not frontend code and nodejs is not supporting ES6. So for now we can only use require(), because "import" is a ES6  syntax. And I also find that eventually "import" in ES6 is converted to require() before executing in the browser.

Reply:
some references:
https://stackoverflow.com/questions/31354559/using-node-js-require-vs-es6-import-export
http://voidcanvas.com/import-vs-require/
http://researchhubs.com/post/computing/javascript/nodejs-require-vs-es6-import-export.html
https://codebrains.io/setting-up-express-with-es6-and-babel/

As you said, node does not support ES6 itself, but it's possible to use Babel to achieve this.  And the performance maybe a little better.